index.d.ts

```diff
@@ -1,4 +1,4 @@
-import Electron from "electron";
+import "electron";
```

Comment:
I think it should be something like
```js
import Electron = require("electron");
```
[link](https://github.com/Microsoft/TypeScript/issues/7185#issuecomment-421632656)

Reply:
It can also be:
```javascript
import * as electron from 'electron'; 
```
But at the end, the import here is just to link the types, there is no real execution of any electron code in this file. So as long as it compiles, it should be good.